pages/button-dropdown/item-element.permutations.page.tsx

```diff
@@ -149,7 +175,7 @@ export default function () {
         <PermutationsView
           permutations={permutations}
           render={permutation => (
-            <Box margin={{ bottom: 'xxl' }} className={styles['dropdown-permutation']}>
+            <Box margin={{ bottom: 'l' }} className={styles['dropdown-permutation']}>
```

Comment:
Can we remove this change please? It makes the diff to permutations hard to review

Reply:
Hmm not sure why I even changed that, removed it.